test_conformance/subgroups/subhelpers.h

```diff
@@ -452,25 +1373,57 @@ struct test
         memset(&idata[0], 0, input_array_size * sizeof(Ty));
```

Comment:
Shouldn't sgmap be memset here as well, since it gets accessed in the kernel for computing get_sub_group_local_id?